runtime/compiler/env/VMJ9.cpp

```diff
@@ -3094,6 +3094,12 @@ bool TR_J9VMBase::supressInliningRecognizedInitialCallee(TR_CallSite* callsite,
                dontInlineRecognizedMethod = true;
                break;
                }
+         case TR::com_ibm_jit_JITHelpers_findElementFromArray:
+            if (comp->cg()->getSupportsInlineStringIndexOf())
+            {
```

Comment:
Indented braces needed to stay consistent with the rest of the codebase.

Reply:
8ffd466